### PR TITLE
test for pull request

### PR DESCRIPTION
**Pull request by AjanZhong**
test for pull request

### CODE REVIEW

**Comment by AjanZhong** [comment]
Add comments here.

**Comment by AjanZhong** [comment]
This is a test, will not be merged!!

File: app/app.c [hunk 1]
```diff
@@ -24,6 +24,7 @@
 #include <app.h>
 #include <kernel/thread.h>
 
+
 extern const struct app_descriptor __apps_start;
```
Comment (bingzhux) [inline app/app.c:27]: wrong.

**Comment by AjanZhong** [comment]
reopen to test git ammend

**Pull request closed without merging**